README.md

```diff
@@ -192,10 +192,20 @@ shares (`dist_key.private`) together with the participants specified in
 
 Once the DKG phase is done, the distributed public key is saved in the local directoryas well as in the configuration folder (`$HOME/.drand` by default) under the file `groups/dist_key.public`.
 
+#### Downloading the distributed public key
+
+To get the distributed public key of a drand protocol run, execute :
```

Comment:
`distributed public key from a drand node...`
You can also add stg like `Please note that drand currently only handle one distributed public. To participate in multiple "drand networks", you need to run different instances of drand.